docs/intro.md

```diff
@@ -1,52 +1,51 @@
-# Quick introduction to xef.ai (Kotlin version)
+## Getting the libraries
 
-After adding the library to your project
-(see the [main README](https://github.com/xebia-functional/xef/blob/main/README.md) for instructions),
-you get access to the `ai` function, which is your port of entry to the modern AI world.
-Inside of it, you can _prompt_ for information, which means posing the question to an LLM
-(Large Language Model). The easiest way is to just get the information back as a string.
+Libraries are published in Maven Central. You may need to  add that repository explicitly
+in your build, if you haven't done it before. Then add the library in the usual way.
 
 ```kotlin
-import com.xebia.functional.xef.auto.*
+repositories {
+    mavenCentral()
+}
 
-fun books(topic: String): List<String> = ai {
-    promptMessage("Give me a selection of books about $topic")
-}.getOrThrow()
+dependencies {
+    implementation("com.xebia:xef-core:<version>")
+    implementation("com.xebia:xef-openai:<version>")
+}
 ```
 
-> **Note**
-> By default the `ai` block connects to [OpenAI](https://platform.openai.com/).
-> To use their services you should provide the corresponding API key in the `OPENAI_TOKEN`
-> environment variable, and have enough credits.
+We publish all libraries at once under the same version, so
+[version catalogs](https://docs.gradle.org/current/userguide/platforms.html#sec:sharing-catalogs)
+could be useful.
 
-In the example above we _execute_ the `ai` block with `getOrThrow`, that throws an exception
-whenever a problem is found (for example, if your API key is not correct). If you want more
-control, you can use `getOrElse` (to which you provide a custom handler for errors), or
-`toEither` (which returns the result using 
-[`Either` from Arrow](https://arrow-kt.io/learn/typed-errors/either-and-ior/)).
+By default, the `OpenAI.conversation` block connects to [OpenAI](https://platform.openai.com/).
+To use their services you should provide the corresponding API key in the `OPENAI_TOKEN`
+environment variable, and have enough credits.
 
-In the next examples we'll write functions that rely on `ai`'s DSL functionality, 
-but without actually extracting the values yet using `getOrThrow` or `getOrElse`.
-We'll eventually call this functions from an `ai` block as we've shown above, and
-this allows us to build larger pipelines, and only extract the final result at the end.
+```shell
+env OPENAI_TOKEN=<your-token> <gradle-command>
```

Comment:
should we include here the option in which we pass the token directly?

Reply:
@Montagon I don't know if is necessary in this point, but we can add it if you see that is interesting

Reply:
It was just an idea, it's fine for now. thanks!